material/admin/templates/admin/change_list_results.html

```diff
@@ -8,7 +8,17 @@
                 {% for header in result_headers %}
                 {% if 'action-checkbox' not in header.class_attrib %}
                 <th scope="col" {{ header.class_attrib }}>
-                    {{ header.text|capfirst }}
+                    {% if header.sortable %}
+                        {% if header.sort_priority == 0 %}
+                            <a href="{{ header.url_primary }}">{{ header.text|capfirst }}</a>
+                        {% elif header.ascending %}
+                            <a href="{{ header.url_primary }}" title="{% trans "Toggle sorting" %}">{{ header.text|capfirst }}</a>&nbsp;<i class="zmdi zmdi-chevron-up"></i>
```

Comment:
https://github.com/viewflow/django-material/blob/master/material/static/material/css/forms.css#L5

this line and zmi class changes the header height, this need to be fixed somehow.